drafts/use-cases/1.1.security-considerations.md

```diff
@@ -10,23 +10,30 @@ There are several possibilities here. One of them would be to use a *@graph* not
 
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@graph": [
         {
             "@id": "/api",
-            "operation": [
-                {
-                    "@type": "hydra:Operation",
-                    "title": "List events",
-                    "method": "GET",
-                    "returns": "hydra:Collection"
-                },
-                {
-                    "@type": "hydra:Operation",
-                    "title": "Create new event",
-                    "method": "POST",
-                    "expects": "schema:Event"
-                }
-            ]
+            "@type": "hydra:EntryPoint",
+            "#events": {
```

Comment:
@alien-mcl: All I can say is how I reacted myself and how I experience others react to the JSON-LD syntax. To programmers, all syntactic sugar comes with a bit of overhead. It needs to be visually parsed and understood, then it takes quite a bit of time getting used to.

Learning the syntax of Ruby and Perl is therefore more taxing than learning the syntax of Lisp, for instance. Note that learning the syntax is different from learning the language. Learning the language (keywords, statements, structure) is not the same as learning its syntax.